bazel/repositories.bzl

```diff
@@ -47,11 +47,16 @@ def proxy_wasm_cpp_host_repositories():
     http_archive(
         name = "wamr",
         build_file = "@proxy_wasm_cpp_host//bazel/external:wamr.BUILD",
-        sha256 = "fb2ca3b0322becb8c62de626181ef66a7be15e36000913f0ad4bf51d4619891b",
-        strip_prefix = "wamr_pre_release",
-        url = "https://github.com/lum1n0us/wasm-micro-runtime/releases/download/WAMR-01-29-2021/wamr_pre_release.zip",
+        sha256 = "88cd7145d7098daacfb0da51548327f4e5303d17ea27611cde6f47c6f7b68059",
+        url = "https://github.com/lum1n0us/wasm-micro-runtime/releases/download/WAMR-01-29-2021/source.zip",
```

Comment:
Should we change it to the WAMR officer Repo?

Reply:
has to wait until our integration success and then official repo will approve my modification.